tests/Cache/MockStorageAdapterTest.php

```diff
@@ -66,8 +66,9 @@ public function testFetch()
     public function testSave()
     {
         $request = $this->getRequestMock();
-        $mockStorage = new $this->class($this->tmpDir, [], ['X-Foo']);
```

Comment:
This test should not be altered.

On the other hand, we should create tests which test specifically the naming strategies.